eng/tools/rush-runner.js

```diff
@@ -22,6 +22,19 @@ const reducedDependencyTestMatrix = {
     '@azure/test-utils-perfstress',
     '@azure-tools/test-recorder',
     '@azure/synapse-monitoring'
+  ],
+  'test-utils': [
+    '@azure-rest/purview-account',
+    '@azure-tests/perf-storage-blob',
+    '@azure-tests/perf-data-tables',
+    '@azure/arm-eventgrid',
+    '@azure/ai-text-analytics',
+    '@azure/identity',
+    '@azure/identity-cache-persistence',
+    '@azure/identity-vscode',
+    '@azure/storage-blob',
+    '@azure/template',
+    '@azure/synapse-monitoring'
   ]
```

Comment:
```suggestion  'test-utils': [
    '@azure/test-utils',
    '@azure/test-utils-perfstress',
    '@azure-tools/test-recorder',
    '@azure-tools/test-recorder-new',
    '@azure-rest/purview-account',
    '@azure-tests/perf-storage-blob',
    '@azure-tests/perf-data-tables',
    '@azure/arm-eventgrid',
    '@azure/identity',
    '@azure/identity-cache-persistence',
    '@azure/identity-vscode',
    '@azure/template',
    '@azure/storage-file-share',
    '@azure/ai-text-analytics',
  ]
```

Reply:
We don't need core packages.
Adding 
- test-utils
- perf framework
- recorder
- recorder-new
- 1 azure-rest package
- 1 core-v1 perf-project
- 1 core-v2 perf project
- 1 arm package that uses recorder
- template
- identity - all packages
- 1 core-v1 package
- 1 core-v2 package

Reply:
One more update @praveenkuttappan, thanks for the PR 🤝

Reply:
The list looks good!

Reply:
updated list.